auth_tkt/ticket.py

```diff
@@ -37,7 +37,8 @@ def validate(ticket, secret, ip='0.0.0.0', timeout=7200, encoding='utf-8'):
     if '!' not in raw:
         return False
 
-    raw = raw[32:]
+    hash_length = len(hashlib.new(digest, b'').hexdigest())
```

Comment:
`hashlib.new(digest).digest_size`?

Reply:
Probably not, the digest size tells hash size in bytes, but the hexadecimal string is Unicode.

```python
import hashlib
hash = hashlib.new('md5', b'')

hash.digest_size
>>> 16

len(hash.hexdigest())
>>> 32

Reply:
Almost, see 7732b8f.